agr_java_core/src/main/java/org/alliancegenome/es/util/IndexManager.java

```diff
@@ -454,6 +454,25 @@ public void listRepo(String repo) { // ES Util
             log.info(info.snapshotId() + "[" + print + "] " + end);
         }
     }
+    
+
+    public void cleanSnapShots(String repo, String snapShotName) {
+        List<SnapshotInfo> list = getSnapshots(repo);
+        TreeMap<Date, SnapshotInfo> map = new TreeMap<>();
```

Comment:
Why are you using a TreeMap, you are not making any use of sorting. HashMap should be sufficient.